legacy-service/schemas/users/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var validations = {
+	'complete' : require( './complete' )
```

Comment:
We could add additional things in there like 

```javascript
'id'       : require( './complete' )( 'PersonnelId' ),
'password' : require( './complete' )( 'Password' )
```

For things that will be used in different places.